### ISSUE TITLE
[Bug] Fireflies are not rendered correctly

### ISSUE THREAD

**Issue by DryTaste** [paulevsGitch/BetterNether#507]
### What happened?

Fireflies and some other mobs exhibit visual glitches as can be seen in the screenshot provided below;

![glitch](https://user-images.githubusercontent.com/3638291/150974160-0f447e0d-dcbf-4efb-8571-6dfc1a0d4275.png)

### BetterNether

6.0.9

### BCLib

1.2.5

### Fabric API

0.46.1

### Fabric Loader

0.12.12

### Minecraft

1.18.1

### Other Mods

```shell
Canvas 1.0.2302 (same results with 1.0.2282)
Better end 1.0.1
```

**Comment by DryTaste** [comment]
Update; after some more tests the issue seems to be caused by Canvas, uninstalling this mod removed the glitches for fireflies. 

I have updated the issue to be a compatability problem.

**Comment by paulevsGitch** [comment]
I guess the reason is incompat with Canvas:
> Canvas 1.0.2302

**Comment by DryTaste** [comment]
> I guess the reason is incompat with Canvas:
> 
> > Canvas 1.0.2302

After some testing that seems to be the case.

**Comment by paulevsGitch** [comment]
I think that canvas require custom shaders for such entity rendering, this is also related to #484

**Comment by DryTaste** [comment]
For anyone coming across this issue;

The effects are way less obnoxious when using Iris & Sodium with "complimentary shaders" installed. Especially so if you disable PR support in the shader settings. This seems to be the superior option because you will enjoy greater mod compatability (canvas breaks stuff like immersive portals and smooth water color transitions in BetterEnd) and will also benefit both the emissive textures this mod has to offer and the peformance increase by sodium. 

The only drawback to disabling PR support is that some emissive stuff stops working (like the chorus flowers from BetterEnd). This is a trade off descision you have to make. 

The ideal solution would be to find a shader for Iris + Sodium that enables emissives/bloom but nothing else.

**Comment by quiqueck** [comment]
The next version should fix this for Iris. Canvas needs more work tho, and we will track that in #484

**Comment by quiqueck** [comment]
The Fix should be in 6.0.10. Please re-open this issue if you still have problems after upgrading.
